pkg/northbound/gnmi/v2/get.go

```diff
@@ -154,8 +154,8 @@ func (s *Server) checkOpaAllowed(ctx context.Context, targetInfo targetInfo, con
 		return nil, err
 	}
 
-	targetVersion := filterTargetForURL(modelPlugin.Info.Version)
-	targetType := filterTargetForURL(modelPlugin.Info.Name)
+	targetVersion := filterTargetForURL(modelPlugin.GetInfo().Info.Version)
```

Comment:
I think this function somehow redundant isn't it? GetInfo then info?

Reply:
No... GetInfo returns ModelPluginInfo, which is a record (previously named ModelPlugin) which contains another structure defined by API called ModelInfo. Yes, it looks somewhat odd, but I think we should leave it as is.